File: lib/layer2_core/dark/e_erprimarycontainer_dark.dart
import '../../layer1_raw/design_token.dart';
import '../../layer1_raw/dark/system.dart';
import '../../layer1_raw/dark/standard.dart';
import '../../layer1_raw/dark/blue.dart';
import '../../layer1_raw/dark/green.dart';
import '../../layer1_raw/dark/red.dart';
import '../../layer1_raw/dark/nothing.dart';

class EErprimarycontainerDark {
  static DesignToken getToken(dynamic activeTheme) {
    switch (activeTheme.toString().split('.').last) {
      case 'system': return csyd18;
      case 'standard': return cstd18;
      case 'blue': return cbd18;
      case 'green': return cgd18;
      case 'red': return crd18;
      case 'nothing': return cnod49;
      default: return cstd18;
    }
  }
}

File: android/src/main/kotlin/com/example/design_engine/layer2_core/dark/EErprimarycontainerDark.kt
package com.example.design_engine.layer2_core.dark
import com.example.design_engine.R
import com.example.design_engine.layer3_logic.AppTheme
object EErprimarycontainerDark {
    fun getResId(activeTheme: AppTheme): Int {
        return when (activeTheme) {
            AppTheme.SYSTEM -> R.string.csyd18
            AppTheme.STANDARD -> R.color.cstd18
            AppTheme.BLUE -> R.color.cbd18
            AppTheme.GREEN -> R.color.cgd18
            AppTheme.RED -> R.color.crd18
            AppTheme.NOTHING -> R.color.cnod49
        }
    }
}
